### PR TITLE
chore(deps): update

### PR DESCRIPTION
**Pull request by evilebottnawi**
**What kind of change does this PR introduce?**

update deps + remove mkdirp, because we don't need this package (built-in) in node

**Did you add tests for your changes?**

No

**If relevant, did you update the documentation?**

No need

**Summary**

update + reduce deps

**Does this PR introduce a breaking change?**

No

**Other information**

No

### CODE REVIEW

File: packages/generators/package.json [hunk 1]
```diff
@@ -18,18 +18,17 @@
...
-    "mkdirp": "^1.0.4",
     "yeoman-generator": "^4.12.0"
   },
   "peerDependencies": {
     "webpack": "4.x.x || 5.x.x",
     "webpack-cli": "4.x.x"
   },
   "devDependencies": {
-    "@types/mkdirp": "^1.0.0",
+    "@types/mkdirp": "^1.0.1",
```
Comment (rishabh3112) [inline packages/generators/package.json:28]:
```suggestion
```

**Review by anshumanv — DISMISSED** [review]

**Review by rishabh3112 — APPROVED** [review]
left a suggestion

File: packages/generators/package.json (hunk 1 above)
Comment (evilebottnawi) [reply packages/generators/package.json]: Fixed

**Comment by evilebottnawi** [comment]
I will send a PR with updating typescript + tools in the next PR

**Pull request merged**